src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --second-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.logo_link {
+  display: inline-block;
+}
+
+.logo_moyo {
+  height: 100%;
+  width: 100%;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  margin-right: 50px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: #000;
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_item :hover {
+  color: var(--second-color);
+  text-decoration: solid 2px black;
+}
+
+.nav__link:hover {
```

Comment:
`.nav__link` is not used in your HTML, so this style can be removed

Reply:
removed